test/test_zoltan_partitioner_0_MPI2.cpp

```diff
@@ -25,30 +25,19 @@ TEST_CASE("ZoltanPartitioner: all land", "[zoltan]")
     // Partition grid
     partitioner->partition(*grid);
 
-    int mpi_size;
-    MPI_Comm_size(MPI_COMM_WORLD, &mpi_size);
-    assert(mpi_size <= 2);
-
     int mpi_rank;
     MPI_Comm_rank(MPI_COMM_WORLD, &mpi_rank);
```

Comment:
I appreciate you didn't add this in, but we could use `doctest`s built-in `test_rank` to get the rank and remove the need for using `MPI_Comm_rank` (see [here](https://github.com/doctest/doctest/blob/master/doc/markdown/extensions.md#mpi_test_case) for more info)